### DISCUSSION TITLE
Develop both chips with Arduino | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#831] [Q&A]
# Develop both chips with Arduino | Seeed Studio Wiki

Develop SenseCAP Indicator both chips with Arduino

https://wiki.seeedstudio.com/SenseCAP_Indicator_ESP32_Arduino/

**Comment by djbxxx** [comment]
Hey :) unfortunately i am having big troubles with this. If i unplug and plug it back in 20 times maybe 1 time it works and 19 times it doesnt work. Very strange! Do you have any suggestions what i might try or what I am doing wrong? I bought 2 sensecaps and both dont work...

Best David

**Comment by MarioHudds** [comment]
The D1 Indicator is not really meant for developement in Ardiuno. PlatformIO is your better friend. Seed Studio should have told us Adrduino is not enough for this project.

**Reply by nfs0619** [reply]
Thanks a lot for the feedback! We’ll look into updating the documentation to make things easier to follow.

**Comment by paulnorman49-byte** [comment]
Can Squareline Studio be used with Arduino?

**Reply by 12-Twelve-12** [reply]
Yes, Squareline Studio can be used with the SenseCAP Indicator. You can refer to https://wiki.seeedstudio.com/sensecap_indicator_project/#overview and https://wiki.seeedstudio.com/SenseCAP_Indicator_How_to_Create_your_own_UI/. If you encounter any issues, please feel free to contact our Tech Support.
